tensorflow/compiler/xla/service/llvm_ir/fused_ir_emitter.cc

```diff
@@ -81,7 +81,8 @@ Status FusedIrEmitter::DefaultAction(HloInstruction* hlo) {
 }
 
 Status FusedIrEmitter::HandleConstant(HloInstruction* constant) {
-  unsigned int global_address_space = gpu::GetGlobalMemoryAddressSpace(module_);
+  unsigned int global_address_space =
```

Comment:
nit picking: I see many more usage of `unsigned` than `unsigned int` in XLA code. perhaps we should stick with `unsigned`?